src/components/FilePreview/FilePreview.svelte

```diff
@@ -1,36 +1,39 @@
 <script lang="ts">
+import { Banner } from 'components'
```

Comment:
Since this import is within a component, it's probably better to import the `Banner` component directly (rather than `from 'components'`) to avoid circular dependencies.
```suggestion
import Banner from 'components/Banner.svelte'
```